src/Services/WorksWithDatabases.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace Modelizer\Selenium\Services;
+
+trait WorksWithDatabases
+{
+    /**
+     * Assert that a given where condition exists in the database.
+     *
+     * @param string $table
+     * @param array  $data
+     * @param string $connection
+     *
+     * @return $this
+     */
+    protected function seeInDatabase($table, array $data, $connection = null)
+    {
+        if (is_null($this->app)) {
+            return; // don't run when there is no application
```

Comment:
Should we throw an exception here? To let a user know that application is not been initialize.